ignite-base/package.json

```diff
@@ -76,5 +77,10 @@
       "XMLHttpRequest",
       "FormData"
     ]
+  },
+  "config": {
+    "ghooks": {
+      "pre-commit": "npm run lint"
```

Comment:
should it lint everything or just the files that are staged?  The previous `lint.sh` would just lint staged files.   Open to thoughts.